src/app/shared/pipes/comment-item-user-initials.pipe.ts

```diff
@@ -0,0 +1,12 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { UserResponse } from '@core/store/user.interfaces';
+
+@Pipe({
+  name: 'commentItemUserInitials',
+  standalone: true,
+})
+export class CommentItemUserInitialsPipe implements PipeTransform {
+  transform(value: UserResponse) {
```

Comment:
chyba powinniśmy też wziąć pod uwagę sytuację, kiedy przyjdzie nam jakis undefined  etc. i tez to obsluzyc